src/main/cpp/chassis/SwerveChassis.cpp

```diff
@@ -473,6 +473,24 @@ bool SwerveChassis::IsSamePose()
 
     return isSamePose;
 }
+bool SwerveChassis::IsSamePose(units::length::inch_t distanceThreshold)
+{
```

Comment:
This works, but I think we want this to be IsDone so we aren't waiting for the time. But again works so lets pull it in. Maybe just add a comment?

Reply:
We should make the original method call the new override with the predefined distance.   

Since you tested with this, though, we should probably pull it in.

Reply:
If you agree, you can merge it; otherwise we can talk tomorrow.

Reply:
Yeah merge it in, we can clean all of this up after states